noxfile.py

```diff
@@ -217,6 +224,9 @@ def unit(session, protobuf_implementation):
     if protobuf_implementation == "cpp":
         session.install("protobuf<4")
 
+    if install_bigframes:
+        session.install("bigframes >= 1.17.0")
```

Comment:
Please use the "extras". Don't install this manually.

Reply:
Sounds good. Updated owlbot.py instead